Mail/Views/New Message/ComposeMessageMacosToolbarView.swift

```diff
@@ -0,0 +1,141 @@
+ along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCoreSwiftUI
+import InfomaniakRichHTMLEditor
+import MailCoreUI
+import MailResources
+import Popovers
+import SwiftUI
+
+struct ComposeMessageMacosToolbarView: View {
+    @ObservedObject public var textAttributes: TextAttributes
+    @Binding var isShowingLinkAlert: Bool
+    @Binding var isShowingFileSelection: Bool
+
+    let extras: [EditorToolbarAction] = [
+        .attachment,
+        .link
+    ]
+
+    let textFormats: [EditorToolbarAction] = [
+        .bold,
+        .underline,
+        .italic,
+        .strikeThrough,
+        .cancelFormat
+    ]
+
+    let textItems: [EditorToolbarAction] = [
+        .unorderedList
+    ]
+
+    var allItems: [[EditorToolbarAction]] {
+        [extras, textFormats, textItems]
+    }
+
+    var body: some View {
+        VStack(spacing: 0) {
+            HStack(alignment: .center) {
+                ForEach(allItems, id: \.self) { items in
+                    ForEach(items, id: \.self) { item in
+                        Button(
+                            action: {
+                                item.action(
+                                    textAttributes: textAttributes,
+                                    isShowingLinkAlert: &isShowingLinkAlert,
+                                    isShowingFileSelection: &isShowingFileSelection
+                                )
+                            }, label: {
+                                item.icon.swiftUIImage
+                                    .padding(IKPadding.medium)
+                            }
+                        )
+                        .buttonStyle(MacosToolbarButtonStyle(isActive: item.isSelected(textAttributes: textAttributes)))
+//                        .popoverToolbarHelp(title: item.accessibilityLabel)
+                    }
+
+                    if allItems.last != items {
+                        Divider()
+                            .padding(.vertical, value: .medium)
+                    }
+                }
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding(.horizontal, IKPadding.composeViewHeaderHorizontal)
+
+            IKDivider()
+        }
+    }
+}
+
+struct MacosToolbarButtonStyle: ButtonStyle {
+    @State private var isHovered = false
+    let isActive: Bool
+
+    func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .background(isHovered ? MailResourcesAsset.hoverMenuBackground.swiftUIColor : .clear)
+            .background(isActive ? MailResourcesAsset.hoverMenuBackground.swiftUIColor : .clear)
+            .foregroundColor(isActive ? .primary : .secondary)
+            .cornerRadius(8)
+            .scaleEffect(configuration.isPressed ? 0.95 : 1.0)
+            .onHover { hovering in
+                withAnimation(.easeInOut(duration: 0.25)) {
```

Comment:
A default animation would be better here. So the animation depends on the environment